launch/arm_moveit_node.launch

```diff
@@ -38,7 +38,7 @@
   <node pkg="hebi_cpp_api_examples" type="arm_moveit_node" name="arm_moveit_node" output="screen"/>
 
   <!-- Start gripper node as appropriate -->
-  <group if="$(eval gripper_type == 'parallel')">
+  <group ns="gripper" if="$(eval gripper_type == 'parallel')">
```

Comment:
This reminds me -- we should also update the launch file (and included parameters) for the `rosie_teach_repeat` launch file

Reply:
I've updated the launchfiles to include from the other param yamls. Also switched the family names from HEBI to Arm, since that seemed to be the consensus on Slack.